versions/1.1/schema/digitized_audioreeldigital.json

```diff
@@ -260,8 +260,9 @@
               "enum": [
                 "stereo",
                 "mono",
-                "unknown",
-                "multitrack"
+                "multitrack",
+                "multi-track",
```

Comment:
okay, as long as its like that on confluence and the spreadsheet, we'll go ahead an kill multitrack as a term in v2